src/client/pps/pps.proto

```diff
@@ -58,6 +58,7 @@ enum JobState {
   JOB_SUCCESS = 3;
   JOB_KILLED = 4;
   JOB_MERGING = 5;
+  JOB_EGRESS = 6;
```

Comment:
Can this be `JOB_EGRESSING`? Our other states that represent ongoing operations use present participle, this one should too.